roles/parse_operator_bundle/templates/parsed_operator_data.yml.j2

```diff
@@ -13,6 +13,7 @@ operator_allnamespaces_support: {{ operator_allnamespaces_support }}
 operator_ownnamespace_support: {{ operator_ownnamespace_support }}
 operator_singlenamespace_support: {{ operator_singlenamespace_support }}
 operator_multinamespace_support: {{ operator_multinamespace_support }}
+operator_valid_subscription: {{ operator_valid_subscription }}
```

Comment:
Have you consider omitting `operator_valid_subscription` the same as for `crd_paths` few lines after?
IMHO relying on default value "Not found'" is not ideal.

Reply:
Good suggestion

Reply:
thank you for changes